torch/_C/__init__.pyi.in

```diff
@@ -1440,7 +1440,7 @@ class _TensorBase(metaclass=_TensorMeta):
 def _multiprocessing_init() -> None: ...
 
 # Defined in torch/csrc/mps/Module.cpp
-def _mps_synchronize() -> None: ...
+def _mps_deviceSynchronize() -> None: ...
```

Comment:
Can you please elaborate why this change is needed? Also, it makes it different from similar `_cuda_syncrhonize`, isn't it?

Reply:
As I said `[here](https://github.com/pytorch/pytorch/pull/100661#discussion_r1186261524)`, the API `synchronizeEvent()` is coming in a separate PR.  So `_mps_synchronize()` alone doesn't imply what it's synchronizing (event, single stream, device, etc.). So i renamed it to clarify what it's synchronizing.